src/bindings/python/fluxacct/accounting/project_subcommands.py

```diff
@@ -134,6 +136,8 @@ def list_projects(conn, cols=None, table=False):
 
         # initialize AccountingFormatter object
         formatter = fmt.AccountingFormatter(cur)
+        if format_string != "":
```

Comment:
why not make `format_string` default to `None`? And then have this check be `if format_string is not None`